webrtc/videoroom/config/runtime.exs

```diff
@@ -66,11 +79,23 @@ end
 
 # stun_servers: "addr:port"
 # turn_servers: "addr:port:username:password:proto"
-config :membrane_videoroom_demo,
-  stun_servers:
-    System.get_env("STUN_SERVERS", "64.233.163.127:19302") |> ConfigParser.parse_stun_servers(),
-  turn_servers: System.get_env("TURN_SERVERS", "") |> ConfigParser.parse_turn_servers(),
-  max_display_num: System.get_env("MAX_DISPLAY_NUM", "3") |> ConfigParser.parse_max_display_num()
+app_envs =
+  [
+    stun_servers:
+      System.get_env("STUN_SERVERS", "64.233.163.127:19302") |> ConfigParser.parse_stun_servers(),
+    turn_servers: System.get_env("TURN_SERVERS", "") |> ConfigParser.parse_turn_servers(),
+    max_display_num:
+      System.get_env("MAX_DISPLAY_NUM", "3") |> ConfigParser.parse_max_display_num()
+  ] ++
+    case System.get_env("MAX_PARTICIPANTS_NUM") do
```

Comment:
What about a default value? It will simplify the case statement here. I know we can't handle many participants right now but 25 as a number seems reasonable?

Reply:
We decided not to set any limit by default. It's more like "if you want to set any limit you can do it but by default there no limitations"